e2e/libs/volume/crd.py

```diff
@@ -367,3 +367,27 @@ def validate_volume_replicas_anti_affinity(self, volume_name):
         for replica in replica_list:
             node_set.add(replica['status']['ownerID'])
         assert len(replica_list) == len(node_set), f"unexpected replicas on the same node: {replica_list}"
+
+    def update_volume_spec(self, volume_name, key, value):
+        # retry conflict error
+        for i in range(self.retry_count):
+            try:
+                volume = self.get(volume_name)
+                spec = volume['spec']
+                spec[key] = value
+                volume['sepc'] = spec
```

Comment:
There is a typo `sepc` here and seems this line is not required.

Reply:
Updated